### PR TITLE
Report a error if grpc server is not started

### PR DESCRIPTION
**Pull request by dimitry-**
Bug: http://b/168295378

### CODE REVIEW

File: OrbitService/OrbitService.cpp [hunk 1]
```diff
@@ -47,6 +47,11 @@ void OrbitService::Run(std::atomic<bool>* exit_requested) {
   LOG("Starting GRPC server at %s", grpc_address);
   std::unique_ptr<OrbitGrpcServer> grpc_server;
   grpc_server = OrbitGrpcServer::Create(grpc_address);
+  if (grpc_server == nullptr) {
+    ERROR("Unable to start GRPC server.");
```
Comment (dpallotti) [inline OrbitService/OrbitService.cpp:51]:
```suggestion
    ERROR("Unable to start gRPC server");
```

File: OrbitService/OrbitService.cpp [hunk 2]
```diff
@@ -47,6 +47,11 @@ void OrbitService::Run(std::atomic<bool>* exit_requested) {
...
+    return;
+  }
+  LOG("GRPC server is running");
```
Comment (dpallotti) [inline OrbitService/OrbitService.cpp:54]:
```suggestion
  LOG("gRPC server is running");
```

**Review by dpallotti — APPROVED** [review]

**Review by IrinaShkviro — APPROVED** [review]
Thanks a lot Dimitry! Just one question: is it possible to enable grpc log also?

**Comment by dimitry-** [comment]
> Thanks a lot Dimitry! Just one question: is it possible to enable grpc log also?

I think it is but considering that we do not really expect grpc to fail, I decided not to at least for now.

File: OrbitService/OrbitService.cpp (hunk 1 above)
Comment (dimitry-) [reply OrbitService/OrbitService.cpp:51]: done.

File: OrbitService/OrbitService.cpp (hunk 2 above)
Comment (dimitry-) [reply OrbitService/OrbitService.cpp:54]: done.

**Pull request merged**